Cubical/Data/Equality/PropositionalTruncation.agda

```diff
@@ -0,0 +1,33 @@
+{- Propositional truncation using ≡ with paths under the hood -}
```

Comment:
Maybe "Propositional truncation for the inductively defined equality type _≡_, with paths under the hood"?